### ISSUE TITLE
Mic does not work on Edge

### ISSUE THREAD

**Issue by mysticaltech** [Shaunwei/RealChar#154]
Hey @Shaunwei, first of all, super amazing project! Love it.

Just one thing, I'm on Linux fedora latest, and and the mic works great on Chrome, but not Edge. I can select the right mic, it signals listening in the browser, but nothing picked up. So for now using it with Chrome, but would be great to have it work with Edge too.

I wish I knew how to debug this some more, don't hesitate to give me breadcrumbs.

**Comment by pycui** [comment]
Yes I think there is some issue with Microphone on Edge. We will look into this!

**Comment by pycui** [comment]
@LucianoSP Chrome not working is unexpected. Have you made sure you select the correct audio input device when you connect? Also did you use a headphone?
